website/content/en/docs/concepts/node-templates.md

```diff
@@ -160,12 +161,16 @@ spec:
 
 The AMI used when provisioning nodes can be controlled by the `amiFamily` field. Based on the value set for `amiFamily`, Karpenter will automatically query for the appropriate [EKS optimized AMI](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-amis.html) via AWS Systems Manager (SSM). When an `amiFamily` of `Custom` is chosen, then an `amiSelector` must be specified that informs Karpenter on which custom AMIs are to be used.
 
-Currently, Karpenter supports `amiFamily` values `AL2`, `Bottlerocket`, `Ubuntu` and `Custom`. GPUs are only supported with `AL2` and `Bottlerocket`. The `AL2` amiFamily does not support ARM64 GPU instance types unless you specify a custom amiSelector.
+Currently, Karpenter supports `amiFamily` values `AL2`, `Bottlerocket`, `Ubuntu`, `Windows2019`, `Windows2022` and `Custom`. GPUs are only supported with `AL2`, `Bottlerocket`, `Windows2019` and `Windows2022`. The `AL2` amiFamily does not support ARM64 GPU instance types unless you specify a custom amiSelector.
```

Comment:
I don't think we support GPUs for EKS Windows yet.

Reply:
Oh, I had checked EC2 Windows support and assumed we also support GPUs. Will remove this change.

Reply:
@tzifudzi to confirm whether pods requesting GPUs work on a Windows node

Reply:
So Windows EKS Windows Optimized AMIs do not currently ship with NVIDIA GPU drivers installed. So this means that pods can be scheduled on the node but the GPU workloads will not be successfully executed unless the appropriate drivers are installed. As such, I will be removing the statement regarding Windows GPU support until EKS Windows ships with drivers installed.